library/src/main/java/com/google/android/exoplayer/metadata/frame/GeobFrame.java

```diff
@@ -13,26 +13,42 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.google.android.exoplayer.metadata;
+package com.google.android.exoplayer.metadata.frame;
 
 /**
  * A metadata that contains parsed ID3 GEOB (General Encapsulated Object) frame data associated
  * with time indices.
  */
-public final class GeobMetadata {
+public final class GeobFrame extends Id3Frame {
 
-  public static final String TYPE = "GEOB";
+  public static final String ID = "GEOB";
 
-  public final String mimeType;
-  public final String filename;
-  public final String description;
-  public final byte[] data;
+  private final String mimeType;
```

Comment:
Why have you made all of these private? We have a preference for making fields public if they're final, rather than having getter methods. Ditto in other files (including the new BinaryFrame file).